slackIntegration-server/src/main/scala/com/fpd/teamcity/slack/SlackGateway.scala

```diff
@@ -121,7 +121,7 @@ class SlackGateway(val configManager: ConfigManager, logger: Logger) {
     }
 
   private def channelChatConfiguration =
-    SlackChatConfiguration.getConfiguration.withName(configManager.senderName.getOrElse(Strings.channelMessageOwner))
+    if (configManager.senderName.isEmpty) SlackChatConfiguration.getConfiguration.asUser() else SlackChatConfiguration.getConfiguration.withName(configManager.senderName.get)
```

Comment:
The better solution is to write another way

```scala
  private def channelChatConfiguration =
    configManager.senderName match {
      case Some(senderName) ⇒
        SlackChatConfiguration.getConfiguration.withName(senderName)
      case _ ⇒
        SlackChatConfiguration.getConfiguration.asUser()
    }
```

Reply:
Thanks, I have updated the code and added the failed tests to the commit message. 

I think they are failing due to windows line endings.

Ian